subscriptions.md

```diff
@@ -0,0 +1,297 @@
+        },
+        "ePerson": {
+          "href": "http://localhost:8080/server/api/core/subscriptions/60/ePerson"
+        },
+        "self": {
+          "href": "http://localhost:8080/server/api/core/subscriptions/60"
+        }
+      }
+    }
+  ],
+  "_links": {
+    "self": {
+      "href": "http://localhost:8080/server/api/core/subscriptions/search/findByEPerson?id=49e5ff01-5467-4acd-be21-9ad7374be929"
+    }
+  },
+  "page": {
+    "size": 20,
+    "totalElements": 1,
+    "totalPages": 1,
+    "number": 0
+  }
+}
+```
+Status codes:
+* 200 OK - if the operation succeed
+* 400 Bad Request - if the dspace_object_id or eperson_id parameters are missing or invalid
+* 401 Unauthorized - if you are not authenticated
+* 403 Forbidden - if you are not logged in with sufficient permissions. Only system administrators or the user specified in the eperson_id parameter can use the endpoint
+
+## Creating a subscription
+**POST /api/core/subscriptions?eperson_id=<:uuid>&dspace_object_id=<:uuid>**
+
+Request body:
+```json
+{
+  "subscriptionType": "Test",
+  "subscriptionParameterList": [
+    {
+      "name": "Frequency",
+      "value": "Daily"
+    },
+    {
+      "name": "Frequency",
+      "value": "WEEKLY"
+    }
+  ]
+}
+```
+The parameters are:
+eperson_id: (mandatory), the uuid of the eperson that will be grant of the subscription.
+dspace_object_id: (mandatory), the uuid of the resource target of the subscription
+
+Return codes:
+* 200 OK - if the operation succeed, the created subscription is returned
+* 400 Bad Request - if one or both the eperson_id and the eperson_id parameters are missing or aren't uuid
+* 401 Unauthorized - if you are not authenticated
+* 403 Forbidden - if you are not logged in with sufficient permissions. Only system administrators and Owner can create subscription
+
+## Updating subscription
+** PUT /api/core/subscription/<:id>**
+
+It is possible to update a subscription with id
+`curl -X PUT '{dspace7-url}/api/core/subscriptions/{id}?dspace_object_id={id}&eperson_id={id}
+' -H "Authorization: Bearer ..." -H 'Content-Type: application/json'
+
+```json
+{
+  "subscriptionType": "Test",
+  "subscriptionParameterList": [
+    {
+      "name": "Frequency",
+      "value": "Weekly"
+    }
+  ]
+}
+```
+
+Return codes:
+* 200 OK - if the operation succeed, the created subscription is returned
+* 400 Bad Request - if one or both the eperson_id and the eperson_id parameters are missing or aren't uuid
+* 401 Unauthorized - if you are not authenticated
+* 403 Forbidden - if you are not logged in with sufficient permissions. Only system administrators and Owner can create subscription
+
+## Deleting a Subscription Object
+**DELETE /api/core/subscriptions/<:id>**
+
+Deletes a subscription with id.
+
+* 204 No content - if the operation succeed.
+* 401 Unauthorized - if you are not authenticated.
+* 403 Forbidden - if you are not logged in with sufficient permissions. It must be creator or administrator.
```

Comment:
small wording here, you should be the owner of the subscription or an administrator